frontend/integration-tests/views/source-to-image.view.ts

```diff
@@ -20,3 +21,27 @@ export const visitOpenShiftImageStream = async(name: string) => {
 export const selectTestProject = async() => {
   await nsDropdown.click().then(() => browser.actions().sendKeys(testName, Key.ARROW_DOWN, Key.ENTER).perform());
 };
+
+export const visitResource = async(resource: string, appName: string) => {
+  await browser.get(`${appHost}/k8s/ns/${testName}/${resource}/${appName}`);
+};
+
+export const deleteResource = async(resource: string, kind: string, appName: string) => {
+  await visitResource(resource, appName);
+  await crudView.isLoaded();
+  await crudView.actionsDropdown.click();
+  await browser.wait(until.presenceOf(crudView.actionsDropdownMenu), 500);
+  await crudView.actionsDropdownMenu.element(by.partialLinkText('Delete ')).click();
+  await browser.wait(until.presenceOf($('#confirm-action')));
+  await $('#confirm-action').click();
+};
+
+export const displayResources = (resources, appName, appLabel) => _.each(resources, ({kind}, resource) => {
+  it(`displays detail view for new ${kind} instance with expected label`, async() => {
```

Comment:
The description should be definitely kept in the *.scenatio.ts* file. I meant only to move the common content. Sorry for the misunderstanding.